utmap-client/src/components/pages/SingleEventPage.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import {
+	Avatar,
+	Collapse,
+	IconButton,
+	Grid,
+	GridList,
+	makeStyles,
+	Paper,
+	Typography,
+} from '@material-ui/core';
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import CloseIcon from '@material-ui/icons/Close';
+import EditIcon from '@material-ui/icons/Edit';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles(theme => ({ //CSS styles on components
+	box: {
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+		backgroundColor: 'gray',
+	},
+	paper: {
+		width: '35rem',
+		marginTop: theme.spacing(6),
+		paddingTop: theme.spacing(4),
+		paddingBottom: theme.spacing(4),
+		paddingLeft: theme.spacing(4),
+		paddingRight: theme.spacing(4),
+		display: 'flex',
+	},
+	largeIcon: {
+		width: theme.spacing(6),
+		height: theme.spacing(6),
+	},
+	largeAvatar: {
+		background: '#002884',
+		width: theme.spacing(10),
+		height: theme.spacing(10),
+	},
+	row: {
+		width: '100%',
+		display: 'flex',
+		alignItems: 'center',
+	},
+	title: {
+		textAlign: 'center',
+	},
+
+}));
+
+function SingleEventPage({startDate, endDate, title, description, location, sublocation, organizer}) {
+	const classes = useStyles();
+	
+	//strings for the dates and location
+	const startDateString = "Start Date: " + (new Date(startDate)).toLocaleString()
+	const endDateString = "End Date: " + (new Date(endDate)).toLocaleString()
+	const locationString = sublocation + " in " + location
```

Comment:
You could insert the string directly in between the <Typography> tags. For example line 103 could be:  
{sublocation} in {location}  
instead of {locationString}.